pyproject.toml

```diff
@@ -54,3 +55,12 @@ skip-string-normalization = true
 [tool.isort]
 profile = "black"
 known_third_party = ["fastapi", "pydantic", "starlette"]
+
+[tool.mypy]
+plugins = "pydantic.mypy"
+show_error_codes = true
+
+# https://mypy.readthedocs.io/en/stable/config_file.html#using-a-pyproject-toml-file
+[[tool.mypy.overrides]]
+ignore_missing_imports = true
+module = []
```

Comment:
I guess this would be 
```
module = ["spylib.tests.*"]
```
based on the original mypy setting?

Reply:
Mypy seemed fine without ignoring the errors so I left it out. @hillairet do you know why the test module was ignored?

Reply:
I did encounter some typing error about `Box` when I ran the tests the first time. I modified the module there and the error went away. I reverted the change and ran the tests again, the previous typing error didn't come back. So I don't know either.. haha 😕

Reply:
You may need to tell mypy not to use the cache - I've had issues with that before

Reply:
I don't remember the reason to avoid running mypy on the tests.
It'd be better to run mypy on them obviously though.
Right now mypy is happy in the Github action.
@lishanl Maybe you need a `poetry update`? Maybe the proper typing for added to `Box` or something?